### DISCUSSION TITLE
Collection Labels

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#30502] [Ideas]
Players often categorize beatmaps into collections based on their features. It would be great to have a setting in the settings that would allow you to display the collection label directly in the beatmap selection menu, without having to right-click to open the menu.
![image](https://github.com/user-attachments/assets/fe98456e-dadb-459d-9481-70aa970c3675)
(right here)

**Comment by cl8n** [comment]
seems nice to have. since maps can be in multiple collections at once, this would probably only be able to show a few at most before becoming too busy

**Reply by ghost** [reply]
> seems nice to have. since maps can be in multiple collections at once, this would probably only be able to show a few at most before becoming too busy

Yes, I thought about it too. It might make sense to have a small switch in the Manage Collections menu, something like "show in song selection menu", for example. Or show only 1-2 labels whose collections are highest in "Manage Collections" menu (priority).
